stardog/datadog_checks/stardog/__about__.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.1'
+__version__ = "1.0.2"
```

Comment:
Please also update the [CHANGELOG](https://github.com/DataDog/integrations-extras/blob/master/stardog/CHANGELOG.md).